streams/src/main/java/org/apache/kafka/streams/processor/internals/SourceNode.java

```diff
@@ -94,4 +102,7 @@ public String toString(String indent) {
         return sb.toString();
     }
 
+    public TimestampExtractor getTimestampExtractor() {
```

Comment:
I cannot, because the test classes (`TopologyBuilderTest` for example) cannot access the protected method and I cannot test it.